lib/recurly/coupon.ex

```diff
@@ -14,6 +14,8 @@ defmodule Recurly.Coupon do
     field :applies_to_non_plan_charges, :boolean
     field :coupon_code,                 :string
     field :coupon_type,                 :string
+    field :created_at,                  :date_time
+    field :deleted_at,                  :date_time
```

Comment:
I think `created_at`, `updated_at`, and `deleted_at` should each include `read_only: true`.